src/extract/cod.py

```diff
@@ -50,5 +50,22 @@ def get_adm3_snakemake():
 def get_adm3(raw_data_dir: str, output_filename: str):
     filepath_hdx = query_api(HDX_YEMEN_ADDRESS, raw_data_dir,
                              [HDX_YEMEN_FILENAME])[HDX_YEMEN_FILENAME]
+    shutil.move(os.path.join(raw_data_dir, filepath_hdx),
+                output_filename)
+
+
+
+
+HDX_YEMEN_ROAD_ADDRESS = 'yemen-roads'
```

Comment:
Constants should come after the imports. 

Ideally, these parameters should actually come from the config file rather than being constants, however, I know that I was the one who wrote it this way in the firstplace, so not a required change.